docs/administration.mesh-policies.md

```diff
@@ -12,13 +12,18 @@ The following meshSubjects are available:
 - meshLandingZone
 - meshUser/Group
 
-In the administration area, there is an option of 'Policies' in the 'Compliance'
-section of the navigation area. After clicking policies, you can view all policies in your meshStack instance.
+You can see all existing meshPolicies in the **Administration** Area by navigating to **Compliance** > **Policies**.
+If you want to see all detected policy violations, you can go to to **Compliance** > **Policy Violations** in the **Administration** Area or on a customer control plane.
+
+> Creating and Editing meshPolicy triggers an evaluation process of the meshPolicy. This process may take a while. After the process ends, you can see the violations caused by the (updated) meshPolicy in the **Policy Violations**.
 
 ## Create meshPolicies
 
-Policy creation is only possible for a user who has a partner admin role. There is a 'Create Policy' button top
-right on the meshPolicies screen. After clicking on it, there is a redirection to the screen of "Create Policy" where all details need to be filled out.
+Policy creation is only possible for a user who has a partner role and the right permission.
```

Comment:
```suggestion
Policy creation is only possible for a user who has a partner role.
```
d: I think we should talk everytime about the role and not the right permission.